models/pitch.js

```diff
@@ -7,7 +7,8 @@ var pitchSchema = new Schema({
   description: String,
   username: String,
   upvote: Number,
-  downvote: Number
+  downvote: Number,
+  date: { type: Date, default: Date.now }
```

Comment:
We may as well add a date to each project so we can use it to sort, or do other things with it in the future.